Client/Configuration/EcosiaBeta.xcconfig

```diff
@@ -7,6 +7,6 @@
 
 MOZ_BUNDLE_DISPLAY_NAME = Ecosia Beta
 MOZ_BUNDLE_ID = com.ecosia.ecosiaapp.firefox
-CODE_SIGN_ENTITLEMENTS = Client/Entitlements/Ecosia.entitlements
+CODE_SIGN_ENTITLEMENTS = Client/Entitlements/EcosiaBeta.entitlements
```

Comment:
Why the `beta` and not just Ecosia?

Reply:
the beta has different app groups and keychain groups and therefore needs different entitlements. To give you some more context:
EcosiaBeta is the one that we used so far in TestFlight and installs in parallel to the "old" app (com.ecosia.ecosiaapp.firefox)
Ecosia has the same bundle id like the old app and will therefore overinstall it and trigger migration if installed

So far we'll deploy both in parallel to keep existing Beta testers. Once we fully switch we can remove the Beta configs again

Reply:
Makes sense, thanks for the brief 👍